internal/commands/diff.go

```diff
@@ -366,7 +367,11 @@ func doDiff(args []string, config diffCommandConfig) error {
 	case listErr != nil:
 		return listErr
 	case numDiffs > 0:
-		return fmt.Errorf("%d object(s) different", numDiffs)
```

Comment:
Didn't realuze that the current behavior is already to exit non zero. Don't remember how I setup my tooling but I always ignored the exit code and used diff to mostly check what I was gonna deploy.

Reply:
actually I want to make `diff` not exit 1 by default. So first I'm adding a flag to control it and the next minor version will flip this behavior. I didn't want to make a backwards-incompatible change just for this.